### PR TITLE
Add time logging to runReplicationControllerTest to help debug timeouts in the future

### PR DESCRIPTION
**Pull request by davidopp**
ref/ #18857

### CODE REVIEW

File: cmd/integration/integration.go [hunk 1]
```diff
@@ -435,11 +435,14 @@ containers:
 }
 
 func runReplicationControllerTest(c *client.Client) {
+	t := time.Now()
 	clientAPIVersion := c.APIVersion().String()
 	data, err := ioutil.ReadFile("cmd/integration/" + clientAPIVersion + "-controller.json")
 	if err != nil {
 		glog.Fatalf("Unexpected error: %v", err)
 	}
+	glog.Infof("Done reading config file, took %v", time.Since(t))
```
Comment (lavalamp) [inline cmd/integration/integration.go]: Don't glog lines include the time? Not that computing the elapsed time hurt anything.

**Comment by lavalamp** [comment]
LGTM--I'll let you self-apply the label on the off chance you don't want to ignore my comment ;)

**Comment by davidopp** [comment]
I think it's worth merging because I added a couple of additional log messages, which serve as intermediate timing points to help narrow down which part is slow. Good point about the log messages including the time, though I'll just leave the time in anyway.

**Comment by davidopp** [comment]
noted test flake in #19999

@k8s-bot unit test this please

**Comment by davidopp** [comment]
Manually merging since it will help debug flake.

**Pull request merged**